run_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
Do we even have bash in the PG Alpine-based image? If so, why not just /bin/bash?

Reply:
It's more universal

Reply:
And less secure.

Reply:
i think security isn't a concern here. I left it as it is for now